<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>430</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>430</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>NUM Alert</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>431</width>
      <height>521</height>
     </rect>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="toolTipDuration">
     <number>-1</number>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Recurring Alerts</string>
     </attribute>
     <widget class="QGroupBox" name="createAlertBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>401</width>
        <height>181</height>
       </rect>
      </property>
      <property name="title">
       <string>Create a Recurring Alert</string>
      </property>
      <widget class="QComboBox" name="alertTypeCombobox">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>111</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Water</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Food</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Stretch</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Walk</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Break</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Custom</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="customMessageLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Custom message</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="customMessageField">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>100</y>
         <width>161</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="addDelayAlertButton">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>140</y>
         <width>121</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Add Alert</string>
       </property>
      </widget>
      <widget class="QLabel" name="alertTypeLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Select an alert type</string>
       </property>
      </widget>
      <widget class="QLabel" name="freqDurationLabel">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>20</y>
         <width>181</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Select a frequency duration (mins)</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="frequencyDurationSpinbox">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>40</y>
         <width>111</width>
         <height>22</height>
        </rect>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>1440</number>
       </property>
      </widget>
      <widget class="QLabel" name="notiDurationLabel">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>80</y>
         <width>191</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Select a notification duration (seconds)</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="notificationDurationSpinbox">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>100</y>
         <width>111</width>
         <height>22</height>
        </rect>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>60</number>
       </property>
      </widget>
     </widget>
     <widget class="QListWidget" name="currentDelayAlertsList">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>220</y>
        <width>401</width>
        <height>171</height>
       </rect>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="currentAlertsLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>200</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Current Recurring Alerts</string>
      </property>
     </widget>
     <widget class="QPushButton" name="deleteDelayAlertButton">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>400</y>
        <width>121</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Delete Selected Alert</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Timed Alerts</string>
     </attribute>
     <widget class="QLabel" name="currentAlertsLabel_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>200</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Current Timed Alerts</string>
      </property>
     </widget>
     <widget class="QListWidget" name="currentTimedAlertsList">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>220</y>
        <width>401</width>
        <height>171</height>
       </rect>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="deleteTimedAlertButton">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>400</y>
        <width>121</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Delete Selected Alert</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="createAlertBox_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>401</width>
        <height>181</height>
       </rect>
      </property>
      <property name="title">
       <string>Create a Timed Alert</string>
      </property>
      <widget class="QComboBox" name="alertTypeCombobox_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>111</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Water</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Food</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Stretch</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Walk</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Break</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Custom</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="customMessageLabel_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Custom message</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="customMessageField_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>100</y>
         <width>161</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="addTimedAlertButton">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>140</y>
         <width>121</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Add Alert</string>
       </property>
      </widget>
      <widget class="QLabel" name="alertTypeLabel_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Select an alert type</string>
       </property>
      </widget>
      <widget class="QLabel" name="freqDurationLabel_2">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>20</y>
         <width>181</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Select a time to occur</string>
       </property>
      </widget>
      <widget class="QLabel" name="notiDurationLabel_2">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>80</y>
         <width>191</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Select a notification duration (seconds)</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="notificationDurationSpinbox_2">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>100</y>
         <width>111</width>
         <height>22</height>
        </rect>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>60</number>
       </property>
      </widget>
      <widget class="QTimeEdit" name="selectTimeBox">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>40</y>
         <width>111</width>
         <height>22</height>
        </rect>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::UpDownArrows</enum>
       </property>
       <property name="currentSection">
        <enum>QDateTimeEdit::HourSection</enum>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Restrictions</string>
     </attribute>
     <widget class="QLabel" name="currentAlertsLabel_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Applications</string>
      </property>
     </widget>
     <widget class="QListWidget" name="applicationsList">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>401</width>
        <height>171</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="deleteDelayAlertButton_2">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>210</y>
        <width>121</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Add to Banned</string>
      </property>
     </widget>
     <widget class="QLabel" name="currentAlertsLabel_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>240</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Banned Applications</string>
      </property>
     </widget>
     <widget class="QListWidget" name="bannedApplicationsList">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>260</y>
        <width>401</width>
        <height>81</height>
       </rect>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="deleteDelayAlertButton_3">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>350</y>
        <width>121</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Unban Selected</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>390</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>test tip</string>
      </property>
      <property name="toolTipDuration">
       <number>4</number>
      </property>
      <property name="text">
       <string>Focus Mode </string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>addDelayAlertButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>add_delayAlert()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>340</x>
     <y>187</y>
    </hint>
    <hint type="destinationlabel">
     <x>240</x>
     <y>-7</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>deleteDelayAlertButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>delete_selected_delayAlert()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>340</x>
     <y>435</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>-12</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addTimedAlertButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>add_timedAlert()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>338</x>
     <y>182</y>
    </hint>
    <hint type="destinationlabel">
     <x>269</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>deleteTimedAlertButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>delete_selected_timedAlert()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>340</x>
     <y>427</y>
    </hint>
    <hint type="destinationlabel">
     <x>352</x>
     <y>-4</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>add_delayAlert()</slot>
  <slot>delete_selected_delayAlert()</slot>
  <slot>add_timedAlert()</slot>
  <slot>delete_selected_timedAlert()</slot>
 </slots>
</ui>
